categories.css

```diff
@@ -1,151 +1,23 @@
-* {
-  margin: 0;
-  padding: 0;
-}
-:root {
-  --home-bg-color: #faeaff;
-  --home-primary-color: #6500b3;
-  --home-secondary-color: #d392ff;
-  --home-contast-color: #2dffd9;
-  --black-color: #000000;
-  --white-color: #ffffff;
-}
-
-main {
-  width: 100%;
-  /* padding: 0 5%; */
-  box-sizing: border-box;
-  position: relative;
-  overflow: hidden;
-}
-
-.navbar {
-  width: 100%;
-  height: 15vh;
-  display: flex;
-  align-items: center;
-  z-index: 2;
-  top: 0;
-  left: 0;
-  background-color: transparent;
-  color: var(--black-color);
-}
-
-.logo {
-  width: 2rem;
-  cursor: pointer;
-  margin-left: 4rem;
-  font-size: 2.5rem;
-  color: var(--home-secondary-color);
-}
-
-nav {
-  margin-left: auto;
-}
-
-nav ul {
-  display: flex;
-}
-
-nav ul li {
-  list-style: none;
-  margin: 0 1rem;
-}
-
-nav ul li a {
-  text-decoration: none;
-  font-size: 1.4rem;
-  color: var(--black-color);
-}
-
-nav ul li a:hover {
-  color: var(--home-secondary-color);
-}
-
-.navsidebar {
-  width: 60%;
-  height: 100vh;
-  top: 0;
-  padding-top: 5rem;
-  position: absolute;
-  left: -200rem;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  transition: all 1s;
-  z-index: 2;
-  /* background-color: var(--home-primary-color); */
-  background-color: transparent;
-}
-
-.navsidebar.active {
-  left: 0;
-}
-
-.navsidebar span {
-  margin: 1rem 0;
-}
-
-.navsidebar span a {
-  text-decoration: none;
-  color: var(--white-color);
-  font-size: 1.3rem;
-}
-
-.navsidebar span a:hover {
-  color: var(--home-bg-color);
-}
-
-.ham-burger {
-  font-size: 2rem;
-  color: var(--home-secondary-color);
-  cursor: pointer;
-  margin-left: auto;
-  display: none;
-}
-
 .body {
-  width: 100%;
-  height: 100vh;
-  display: flex;
-  flex-direction: column;
-  top: 0;
   background-image: linear-gradient(
     45deg,
-    var(--home-contast-color),
-    var(--home-secondary-color)
+    var(--contrast-bg-color),
+    var(--secondary-color)
   );
-  color: var(--white-color);
   background-repeat: no-repeat;
   background-size: cover;
 }
 
-.primary-btn {
-  background-color: var(--home-secondary-color);
-  margin-left: 0;
-  padding: 0.6rem;
-  border: 0;
-  font-size: 1.4rem;
-  margin: 1rem 0;
-  width: fit-content;
-  border-radius: 6px;
-  color: var(--white-color);
-}
-
 .categories-content {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
-  padding: 0 4rem;
-  flex-wrap: wrap;
-  height: 85vh;
+  margin-top: 10%;
```

Comment:
why margin-top in percentage and others are in rem ?

Reply:
bec i want it in according to screen 10% not in rem